draft-ietf-tls-esni.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Encrypted Server Name Indication for TLS 1.3
-abbrev: TLS 1.3 SNI Encryption
+title: Encrypted ClientHello for TLS 1.3
+abbrev: TLS 1.3 Encrypted ClientHello
```

Comment:
TLS Encrypted ClientHello —> TECH!

(Kidding, kidding)

Reply:
Hah!

Reply:
Seeing this, I think that you should remove 1.3 from the title.  Sure, this is 1.3 only, but I think that this design is fine for 1.4 also.  If it isn't, I'm sure we can deal with that eventually.  I don't want to find that we ship a 1.4 (or 4) that is compatible with this and have people tell me that it isn't compatible.

Reply:
That's a fair point. I'll make that change in a separate PR!